bindings.c

```diff
@@ -5874,6 +5878,275 @@ int proc_read(const char *path, char *buf, size_t size, off_t offset,
 	}
 }
 
+static int sys_devices_system_cpu_read(char *buf, size_t size, off_t offset,
+		struct fuse_file_info *fi)
+{
+	struct fuse_context *fc = fuse_get_context();
+	struct file_info *d = (struct file_info *)fi->fh;
+	char *cache = d->buf;
+	ssize_t total_len = 0;
+	if (offset){
+		if (!d->cached)
```

Comment:
- please add a new line after ssiz_t and if (offset)
- please leave a spacer after the closing ) and the opening {